package.json

```diff
@@ -13,7 +13,9 @@
     "styleguide": "cd ./styleguide && yarn dev",
     "styleguide:build": "cd ./styleguide && yarn build:lib && cd ../",
     "test": "jest",
-    "precommit": "yarn lint"
+    "precommit": "yarn lint",
+    "e2e:local": "cypress run --headed",
```

Comment:
I would rather add `cypress run --headed` to our docs.

Reply:
ok, no worries... I probably shouldn't have pushed that file up, it was not intended to make the final cut... I was having issues running `cypress run` locally due to the fact that I had added `cypress-cucumber-preprocessor` globally, but not `cypress`